lua/duplicate/init.lua

```diff
@@ -45,6 +45,9 @@ local duplicate_cur_line = function()
   end
   local last_line = line_nr + line_count - 1
   vim.api.nvim_buf_set_lines(0, last_line, last_line, false, lines)
+  -- Set Cursor to duplicated line
+  local cursor_line, cursor_col = unpack(vim.api.nvim_win_get_cursor(0))
+  vim.api.nvim_win_set_cursor(0, { cursor_line + 1, cursor_col })
```

Comment:
This should be
```suggestion
  vim.api.nvim_win_set_cursor(0, { cursor_line + line_count, cursor_col })
```
I think

Reply:
but wouldn't `line_count` always be one when duplicating the current line?

Reply:
You can now add a count (`vim.v.count`) before `ydd`, e.g. `3ydd` to duplicate the next 3 lines. Calling the function `duplicate_cur_line` is a bit confusing now though.

Reply:
ah, I see. Haven't thought about that one. Alright, added another commit replacing the `1` with `line_count`